Header header
uint8 motor_type
uint8 motor_mode
